es/08_1_Enviando_una_Transaccion_con_Bloqueo_de_Tiempo.md

```diff
@@ -1,22 +1,22 @@
-# 8.1: Envío de una transacción con un tiempo de bloqueo
+# 8.1: Enviando una transacción con un tiempo de bloqueo
 
 Los capítulos anteriores mostraron dos formas diferentes de enviar fondos desde múltiples máquinas y a múltiples destinatarios. Pero hay otras dos formas de cambiar fundamentalmente las transacciones básicas. El primero de ellos es variar el tiempo eligiendo un tiempo de bloqueo. Esto le brinda la posibilidad de enviar transacciones sin procesar en algún momento en el futuro.
 
 ## Entender como funcionan los bloqueos de tiempo
 
 Cuando crea una transacción de tiempo de bloqueo, la bloquea con un número que representa la altura de un bloque (si es un número pequeño) o una marca de tiempo UNIX (si es un número grande). Esto le dice a la red Bitcoin que la transacción no se puede poner en un bloque hasta que haya llegado la hora especificada o la cadena de bloques haya alcanzado la altura especificada.
 
->:book: _¿Cuál es la altura del bloque? _ Es el recuento total de bloques en la cadena, volviendo al bloque génesis de Bitcoin.
+>:book: _¿Que es la altura de bloque?_ Es el recuento total de bloques en la cadena, remontandose al bloque génesis/inicial de Bitcoin.
```

Comment:
@javiervargas Sorry, I didn't mean to put both options in text, I was just proposing a different translation. Just keep it as it was.